docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, the period can be one of the values of `Tizen.Applications.AlarmWeekFlag` the value will be phase-aligned with other period of alarm.
```

Comment:
According to https://github.com/Samsung/tizen-docs/pull/754#discussion_r297464328, we will use "Since". 
Doubt:
Can we use "time period" instead of "period" to be more clear and rephrase as:

Since Tizen 6.0, the time period of an alarm can be one of the values of `Tizen.Applications.AlarmWeekFlag`. 

Please verify and share your suggestion on the same

The second part of the sentence is a little ambiguous.
Are we trying to say:
This time period value will be phase-aligned with another time period value of the alarm.

It will be good if you can elaborate further.
Request you to verify and elaborate on the sentence so that I can rephrase it better.

Reply:
I'm sorry about the word 'Tizen.Applications.AlarmWeekFlag'
It should be 'Tizen.Applications.AlarmStandardPeriod'

Please review again.

Reply:
OK. I will.